From e8a179161bbe70ba6bbe2f0dc2f5882ce4e2eafe Mon Sep 17 00:00:00 2001
From: Bruno Martins <bgcngm@gmail.com>
Date: Mon, 3 Sep 2018 13:27:12 +0200
Subject: [PATCH 18/50] NavigationBarView: Avoid NPE before mPanelView is
 created

 * Caught during initial setup, while SetupWizard is still running

Change-Id: I013f1a06812fea34b5201f1f3c1f3e11b1c4c3e1
---
 .../android/systemui/statusbar/phone/NavigationBarView.java    | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/packages/SystemUI/src/com/android/systemui/statusbar/phone/NavigationBarView.java b/packages/SystemUI/src/com/android/systemui/statusbar/phone/NavigationBarView.java
index 180402e4803d..e9b24b9c895f 100644
--- a/packages/SystemUI/src/com/android/systemui/statusbar/phone/NavigationBarView.java
+++ b/packages/SystemUI/src/com/android/systemui/statusbar/phone/NavigationBarView.java
@@ -780,7 +780,8 @@ public class NavigationBarView extends FrameLayout implements PluginListener<Nav
     }
 
     private void updateSlippery() {
-        setSlippery(!isQuickStepSwipeUpEnabled() || mPanelView.isFullyExpanded());
+        setSlippery(!isQuickStepSwipeUpEnabled() ||
+                (mPanelView != null && mPanelView.isFullyExpanded()));
     }
 
     private void setSlippery(boolean slippery) {
-- 
2.17.1

